release_notes/365-release-notes.adoc

```diff
@@ -0,0 +1,87 @@
+[id="365-release-notes"]
+= Red Hat Advanced Cluster Security for Kubernetes 3.65
+include::modules/common-attributes.adoc[]
+:context: 365-release-notes
+
+toc::[]
+
+{product-title} (RHACS) 3.65 includes feature enhancements, bug fixes, scale improvements, and other changes.
+
+*Release date:* September 6, 2021
+
+[id="new-features"]
+== New features
+
+[id="mitre-attck-framework"]
+=== MITRE ATT&CK framework
+You can now use the link:https://attack.mitre.org/matrices/enterprise/containers/[MITRE ATT&CK Framework] to categorize policies in the {product-title}. When you xref:../operating/manage-security-policies.adoc#create-custom-policies[create custom security policies], you can now add MITRE ATT&CK framework's adversary tactics and techniques related information.
+
+[id=""]
```

Comment:
This empty ID value seems like an oversight, but I'm open to suggestion.  Also, this is the first time that I've *not* seen `_{context}` as a suffix for the ID.  I guess it's up to you, but seems outside the doc guidelines.

Reply:
Similar to OCP release notes https://raw.githubusercontent.com/openshift/openshift-docs/enterprise-4.8/release_notes/ocp-4-8-release-notes.adoc I haven't used the context here.